Percy/lib/PercyAppiumCapabilities.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+
+namespace PercyIO.Appium
+{
+  internal class PercyAppiumCapabilites
+  {
+    private Dictionary<string, object> capabilites;
+    internal PercyAppiumCapabilites(Dictionary<string, object> capabilites)
+    {
+      this.capabilites = capabilites;
+    }
+
+    public Object? getValue(String key)
```

Comment:
```suggestion
    public Object? getValue(String key)
```
Convert this to templated so that dont have to use Object return